distribution/zip/pom.xml

```diff
@@ -60,7 +60,11 @@
                         </goals>
                         <configuration>
                             <target if="${run}">
-                                <ant antfile="${elasticsearch.integ.antfile}" target="start-foreground"/>
+                                <!-- use conventional port numbers -->
+                                <ant antfile="${elasticsearch.integ.antfile}" target="start-foreground">
+                                    <property name="integ.http.port" value="9200"/>
```

Comment:
These are not overridable, sorry. Its not supposed to be configurable: and it generally does not work. I am happy to remove the properties and sprinkle 9400 and 9500 everywhere instead to make that clear.

I am also happy to stop supporting run.sh completely: its getting in the way big time here. I get bugged on a daily basis because someone is running es on 9200 and i'm just trying to make things work better.